package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "mdn-translated-content",
+  "version": "1.0.0",
+  "description": "Translated content of https://developer.mozilla.org",
+  "private": true,
+  "scripts": {
+    "lint:md": "markdownlint \"**/*.md\" -i node_modules"
```

```suggestion
    "lint:md": "markdownlint \"**/*.md\" -i node_modules",
    "fix:md": "npm run lint:md -- --fix"
```
allow contributors to fix errors in local before commits